en/docs/reference/connectors/google-ads-connector/google-ads-configuration.md

```diff
@@ -0,0 +1,725 @@
+        </tr>
+        <tr>
+            <td><code>customerId</code></td>
+            <td>The ID of the customer being queried. Type: string</td>
+            <td>Yes</td>
+        </tr>
+        <tr>
+            <td><code>query</code></td>
+            <td>The query string. Type: string</td>
+            <td>Yes</td>
+        </tr>
+        <tr>
+            <td><code>pageToken</code></td>
+            <td>Token of the page to retrieve. If not specified, the first page of results will be returned. Use the value obtained fromnextPageTokenin the previous response in order to request the next page of results. Type: string</td>
+            <td>No</td>
+        </tr>
+        <tr>
+            <td><code>pageSize</code></td>
+            <td>Number of elements to retrieve in a single page. When too large a page is requested, the server may decide to further limit the number of returned resources. Type: integer</td>
+            <td>No</td>
+        </tr>
+        <tr>
+            <td><code>validateOnly</code></td>
+            <td>If true, the request is validated but not executed. Type: boolean</td>
+            <td>No</td>
+        </tr>
+        <tr>
+            <td><code>returnTotalResultsCount</code></td>
+            <td>If true, the total number of results that match the query ignoring the LIMIT clause will be included in the response. Default is false. Type: boolean</td>
+            <td>No</td>
+        </tr>
+        <tr>
+            <td><code>summaryRowSetting</code></td>
+            <td>Determines whether a summary row will be returned. By default, summary row is not returned. If requested, the summary row will be sent in a response by itself after all other query results are returned. Type: enum [SummaryRowSetting](https://developers.google.com/google-ads/api/rest/reference/rest/v17/SummaryRowSetting)</td>
+            <td>No</td>
+        </tr>
+    </table>
+
+    **Sample configuration**
+
+    ```xml
+    <googleAds.search configKey="GGOGLE_ADS_CONN">
+        <customerId>{json-eval($.customer_id)}</customerId>
+        <query>{json-eval($.query)}</query>
+    </googleAds.search>
+    ```
+ 
+    **Sample request**
+
+    ```json
+    {
+        "customer_id": "123123123",
+        "query": "SELECT customer_client.client_customer FROM customer_client"
+    }
+    ```
+
+## Error codes related to Google Ads Connector
+
+The connector may encounter errors during operation execution. When an error occurs, the `ERROR_MESSAGE` property will contain detailed information about the error. You can handle these errors using a `Fault Sequence` in your integration. For more information, refer to [Using Fault Sequences]({{base_path}}/learn/examples/sequence-examples/using-fault-sequences/).
+
+| Error code | Description |
+| -------- | ------- |
+| 701001 | A general error has occurred. |
+| 701002 | An error has occurred due to an invalid configuration. |
+| 701003 | An error has occurred in the access token generation flow. |
```

Comment:
Shall we reword the description of each error code as follows (for clarity): 
- General error
- Invalid configuration error
- Error in access token generation flow